apps/client/src/sttr_client/models/checker.js

```diff
@@ -50,6 +50,33 @@ class Checker {
     return this._stack;
   }
 
+  /**
+   * Get data from the stack and return a list of questionIds with the given answers.
+   * @returns {Answers[]} -
```

Comment:
This function returns an object with "question.id" as keys and "answer" values.
So `string: (boolean|string|number|string[])` I guess.